src/constants/index.ts

```diff
@@ -343,3 +344,12 @@ export enum FARM_TAB {
   MY_FARMS = 'my_farms',
   VESTING = 'vesting',
 }
+
+export const EIP712Domain = [
+  { name: 'name', type: 'string' },
+  { name: 'version', type: 'string' },
+  { name: 'chainId', type: 'uint256' },
+  { name: 'verifyingContract', type: 'address' },
+]
+
+export const SUPPORT_LIMIT_ORDER = true
```

Comment:
Is it kind of feature flag?

Reply:
we will release and hide UI.

Reply:
Should we have a separated file name as `constants/flag.ts`?

Reply:
I think no need, we will remove it soon